controllers/controllers/workloads/env/builder.go

```diff
@@ -8,17 +8,17 @@ import (
 	korifiv1alpha1 "code.cloudfoundry.org/korifi/controllers/api/v1alpha1"
 	"code.cloudfoundry.org/korifi/controllers/controllers/shared"
 	"code.cloudfoundry.org/korifi/controllers/controllers/workloads"
-	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 type vcapServicesPresenter struct {
-	UserProvided []serviceDetails `json:"user-provided,omitempty"`
+	UserProvided []ServiceDetails `json:"user-provided,omitempty"`
 }
 
-type serviceDetails struct {
+type ServiceDetails struct {
```

Comment:
The `serviceDetails` type has been made public only for the sake of being used in the integration test. That looks a bit smelly to me

Reply:
Yeah this one was a bit odd. We were originally matching the JSON, but because it's a list with no guaranteed order, that failed once we added more than one CFServiceBinding. The only way we got that to work was to unmarshall it into a struct and use gomega's ConsistOf functionality.